manifests/bucketeer/charts/experiment-calculator/values.yaml

```diff
@@ -0,0 +1,93 @@
+replicaCount: 1
+
+image:
+  repository: ghcr.io/bucketeer-io/bucketeer-experiment-calculator
+  pullPolicy: IfNotPresent
+
+fullnameOverride: "experiment-calculator"
+
+namespace: default
+
+env:
+  project:
+  stanHost: localhost
+  stanPort: 8080
+  mysqlUser:
+  mysqlPass:
+  mysqlHost:
+  mysqlPort: 3306
+  mysqlDbName:
+  environmentService: localhost:9001
+  experimentService: localhost:9001
+  eventCounterService: localhost:9001
+  logLevel: info
+  port: 9090
+  metricsPort: 9002
+  traceSamplingProbability: 0.001
+  timezone: UTC
+
+affinity: {}
+
+nodeSelector:
+
+hpa:
+  enabled:
+  minReplicas:
+  maxReplicas:
+  metrics:
+    cpu:
+      targetAverageUtilization:
+    memory:
+      targetAverageUtilization:
+
+httpstan:
+  image:
+    repository: ghcr.io/bucketeer-io/bucketeer-httpstan
+    tag: 0.0.1
+    pullPolicy: IfNotPresent
+  config:
+  port: 8080
+  healthPath: v1/health
+  resources: {}
+
+envoy:
+  image:
+    repository: envoyproxy/envoy-alpine
```

Comment:
Please update the envoy repository and tag version.
https://github.com/bucketeer-io/bucketeer/blob/main/manifests/bucketeer/charts/account-apikey-cacher/values.yaml#L53-L54